.php-cs-fixer.dist.php

```diff
@@ -7,6 +7,11 @@
     \file_get_contents('https://raw.githubusercontent.com/zenstruck/.github/main/.php-cs-fixer.dist.php')
 );
 
+$finder = PhpCsFixer\Finder::create()
+    ->in([__DIR__.'/src', __DIR__.'/tests'])
+    ->notPath('ForceFactoriesTraitUsage')
```

Comment:
for some reasons, I need to keep tests with prefix `test` and not with annotation `@test` in the end to end tests